src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1638,8 +1666,8 @@ private bool FieldsAllSet(int containingSlot, LocalState state)
         {
             Debug.Assert(containingSlot != -1);
             Debug.Assert(!state.IsAssigned(containingSlot));
-            VariableIdentifier variable = variableBySlot[containingSlot];
-            NamedTypeSymbol structType = VariableType(variable.Symbol) as NamedTypeSymbol;
+            VariableIdentifier variable = variableBySlot[containingSlot].Identifier;
+            NamedTypeSymbol structType = (NamedTypeSymbol)VariableType(variable.Symbol).TypeSymbol;
```

Comment:
>.TypeSymbol [](start = 87, length = 11)

Should we use a conditional access here?

Reply:
Done.